roles/sssd/tasks/main.yml

```diff
@@ -39,19 +39,24 @@
   become: true
 
 #
-# Note that /etc/openldap/ldap.conf is not required for SSSD nor to make SSH logins work,
+# Note that /etc/openldap/*.conf is not required for SSSD nor to make SSH logins work,
 # but it is used
 #   * for tools like the ldapsearch command, which may be used for debugging
 #   * for managing users and groups in a local LDAP with roles/regular_users/tasks/ldap_regular_users.yml
 #   * and by scripts on the cluster: e.g. for fetching quota values.
 #
-- name: Deploy /etc/openldap/ldap.conf
+- name: Deploy /etc/openldap/*.conf files
   ansible.builtin.template:
-    src: openldap-ldap.conf
-    dest: /etc/openldap/ldap.conf
+    src: "{{ item.src }}"
+    dest: "/etc/openldap/{{ item.dest }}"
     owner: root
     group: root
     mode: '0644'
+  with_items:
+    - src: openldap-ldap.conf
+      dest: ldap.conf
```

Comment:
Is there a reason on having a template named in one and destination another?
Might be more clear if they are named the same.

Reply:
Yes, there can be also an `ldap.conf` for SSH: same filename and often results in a lot of confusion. By explicitly appending `openldap` to the name for the template, I hope it is clear which one this is.